doc/paq-nvim.txt

```diff
@@ -163,6 +163,13 @@ imported as `paq`, the functions are:
 
     Default value: |paq-dir|
 
+    `opt`
+    Boolean that change if, by default, plugins are ego loaded or lazy loaded. 
```

Comment:
There's a couple of typos here. It should be _changes_ instead of _change_ and _eagerly_ instead of _ego_.